internal/cluster/validator.go

```diff
@@ -258,26 +258,33 @@ func (v *clusterValidator) printIsIngressVipValid(context *clusterPreprocessCont
 // conditions to have a valid number of masters
 // 1. have exactly three masters
 // 2. have less then 3 master but enough auto-assign hosts that can become masters
+// 3. have at least 2 workers, if workers configured
```

Comment:
I think the comment does not match the code - in the code, you check that we don't have 1 worker which is illegal